scripts/results_generation/ebikon_data_scoring.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import json
+import statistics
+from pathlib import Path
+
+from compare import get_score_folder
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--rewrite-gmcc",
```

Comment:
Looking at line 36, is there a typo?
--rewrite_gmcc is the correct one right?

Reply:
argparse automaticall yremap keyword with "-" to "_" so that standard and respected :)